disttask/framework/planner/planner_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package planner_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/ngaut/pools"
+	"github.com/pingcap/tidb/disttask/framework/planner"
+	"github.com/pingcap/tidb/disttask/framework/storage"
+	"github.com/pingcap/tidb/testkit"
+	"github.com/stretchr/testify/require"
+	"github.com/tikv/client-go/v2/util"
+)
+
+type mockPlan struct{}
+
+func (mockPlan) ToTaskMeta() ([]byte, error) {
+	return []byte("mock"), nil
+}
+
+func (mockPlan) FromTaskMeta([]byte) error {
+	return nil
+}
+
+func (mockPlan) ToPhysicalPlan(planner.PlanCtx) (*planner.PhysicalPlan, error) {
+	return &planner.PhysicalPlan{}, nil
+}
+
+func TestPlanner(t *testing.T) {
+	ctx := context.Background()
+	store := testkit.CreateMockStore(t)
+	gtk := testkit.NewTestKit(t, store)
```

Comment:
Seems useless? Bootstrap a testkit cost some resource, better not add it.

Reply:
we need testkit to submit a global task like what we do in handle_test https://github.com/pingcap/tidb/blob/7e66761047b01b62c4097892be14555f83a3c3b3/disttask/framework/handle/handle_test.go#L34